libs/common-system/src/test/java/org/codice/ddf/configuration/PathResolverTest.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.configuration;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.nio.file.Paths;
+
+import org.junit.Test;
+
+public class PathResolverTest {
+
+    @Test
+    public void testNullPath() {
+        System.clearProperty("ddf.home");
+        assertThat(new PathResolver(null).getPath(), equalTo(null));
+    }
+
+    @Test
+    public void ddfHomeNotSet() {
+        System.clearProperty("ddf.home");
+        assertThat(new PathResolver("test/path").getPath(),
+                equalTo(Paths.get(System.getProperty("user.dir"), "test/path")
+                        .toString()));
+    }
+
+    @Test
+    public void testTransformedPath() {
+        System.setProperty("ddf.home", "/absolute/");
```

Comment:
Does our `ddf.home` property normally end with a tailing slash? That doesn't look right.